doc/source/running.rst

```diff
@@ -63,12 +61,12 @@ Running |pbm.app| Commands
 Configuring a Remote Store for Backup and Restore Operations
 --------------------------------------------------------------------------------
 
-This must done once, at installation time, before backups can be listed, made,
-or restored. Please see :ref:`pbm.config`.
+This must done once, at installation or re-installaton time, before backups can
```

Comment:
This must be done (note missing `be')
For the sake of simlicity: `during installation or re-installation`; before listing, making, or restoring backups